ClickUpApp.ts

```diff
@@ -28,7 +30,7 @@ export class ClickUpApp extends App {
         super(info, logger, accessors);
     }
 
-    private oauth2ClientInstance: IOAuth2Client;
+    public oauth2ClientInstance: IOAuth2Client;
```

Comment:
Why did you change this to `public` from `private`?

Maybe you should consider creating a public get method to expose this private prop, something like:
```

private oauth2Config......

public getOauth2Config() {
  return this.oauth2Config;
}
```

Reply:
I wasting testing it and mistakenly left it public, I implemented getOauth2Config() already below line 102.